src/ocm/components/clusterConfiguration/DiscoveryImageForm.tsx

```diff
@@ -6,27 +6,30 @@ import { getApiErrorMessage, handleApiError } from '../../api';
 import {
   Cluster,
   CpuArchitecture,
-  DiscoveryImageConfigForm,
   DiscoveryImageFormValues,
   ErrorState,
   LoadingState,
+  StatusErrorType,
 } from '../../../common';
 import { forceReload, updateCluster } from '../../reducers/clusters';
 import useInfraEnv from '../../hooks/useInfraEnv';
 import { DiscoveryImageFormService } from '../../services';
+import { OcmDiscoveryImageConfigForm } from './OcmDiscoveryImageConfigForm';
 
 type DiscoveryImageFormProps = {
   cluster: Cluster;
   onCancel: () => void;
   onSuccess: () => Promise<void>;
   cpuArchitecture: CpuArchitecture;
+  onSuccessIpxe: () => Promise<void>;
```

Comment:
The IPXE option has been added also to `Day2GenerateISO` but the prop has not been defined there, so the script generation is not working for Day2.

Reply:
Ok, I'm adding the iPXE option to Day2 also.